Source/Config/Frames/Selling.xml

```diff
@@ -37,32 +37,57 @@
           <Anchor point="TOPLEFT" relativeKey="$parent.AuctionChatLog" relativePoint="BOTTOMLEFT" />
         </Anchors>
       </Frame>
-      <Frame inherits="AuctionatorConfigurationCheckbox" parentKey="AltClick">
-        <KeyValues>
-          <KeyValue key="labelText" value="AUCTIONATOR_L_CONFIG_ALT_CLICK" type="global" />
-          <KeyValue key="tooltipTitleText" value="AUCTIONATOR_L_CONFIG_ALT_CLICK_TOOLTIP_HEADER" type="global" />
-          <KeyValue key="tooltipText" value="AUCTIONATOR_L_CONFIG_ALT_CLICK_TOOLTIP_TEXT" type="global"/>
-        </KeyValues>
+      <Frame inherits="AuctionatorDropDown" parentKey="BagSelectShortcut">
+        <Scripts>
+          <OnLoad>
+            self.text = {"Right Click", "Alt + Left Click", "Alt + Right Click", "Shift + Left Click", "Shift + Right Click", "Disabled"}
+            self.values = {"right click", "alt left click", "alt right click", "shift left click", "shift right click", "none"}
+            self.labelText = AUCTIONATOR_L_CONFIG_SELLING_BAG_SELECT_SHORTCUT
+
+            self:AuctionatorOnLoad()
+          </OnLoad>
+        </Scripts>
+        <Anchors>
+          <Anchor point="TOPLEFT" relativeKey="$parent.PriceHistory" relativePoint="BOTTOMLEFT"/>
+        </Anchors>
+      </Frame>
+      <Frame inherits="AuctionatorDropDown" parentKey="CancelShortcut">
+        <Scripts>
+          <OnLoad>
+            self.text = {"Left Click", "Right Click", "Alt + Left Click", "Alt + Right Click", "Shift + Left Click", "Shift + Right Click", "Disabled"}
+            self.values = {"left click", "right click", "alt left click", "alt right click", "shift left click", "shift right click", "none"}
+            self.labelText = AUCTIONATOR_L_CONFIG_SELLING_CANCEL_SHORTCUT
+
+            self:AuctionatorOnLoad()
+          </OnLoad>
+        </Scripts>
         <Anchors>
-          <Anchor point="TOPLEFT" relativeKey="$parent.PriceHistory" relativePoint="BOTTOMLEFT" />
+          <Anchor point="TOPLEFT" relativeKey="$parent.BagSelectShortcut" relativePoint="BOTTOMLEFT"/>
         </Anchors>
       </Frame>
-      <Frame inherits="AuctionatorConfigurationCheckbox" parentKey="ShiftCancel">
+      <Frame inherits="AuctionatorDropDown" parentKey="BuyShortcut">
+        <Scripts>
+          <OnLoad>
+            self.text = {"Left Click", "Right Click", "Alt + Left Click", "Alt + Right Click", "Shift + Left Click", "Shift + Right Click", "Disabled"}
```

Comment:
And here... I wonder if these might make sense as separate components?

Reply:
I was trying to avoid having a component for each drop down (would have liked to use KeyValues to put the values in, but arrays aren't supported).

Would a `MouseButtonChoiceDropDown` just to get the code into a separate file work?